File: FrontEnd_APK_PacienteApp/src/app/_models/paciente.ts
export class PacienteVO
{
    carnetDeIdentidad:any;
    nombres:any;
    apellidos:any;
    lugarDeNacimiento:any;
    fechaDeNacimiento:any;
    sexo:any;
    numeroDeContacto:any;
    correoElectronico:any;
    contrasena:any;
    nombreDeUsuario:any;
    parientes:any;
    hash:any;
    agendaVirtual:any;
    token: String|any;
    
    constructor(carnetDeIdentidad:any,nombres:any,apellidos:any,lugarDeNacimiento:any,fechaDeNacimiento:any,sexo:any,numeroDeContacto:any,parientes:any,correoElectronico:any,contrasena:any,nombreDeUsuario:any){
        this.carnetDeIdentidad=carnetDeIdentidad;
        this.nombres=nombres;
        this.apellidos=apellidos;
        this.lugarDeNacimiento=lugarDeNacimiento;
        this.fechaDeNacimiento=fechaDeNacimiento;
        this.sexo=sexo;
        this.numeroDeContacto=numeroDeContacto;
        this.parientes=parientes;
        this.correoElectronico=correoElectronico;
        this.contrasena=contrasena;
        this.nombreDeUsuario=nombreDeUsuario;
    }
    get_carnetDeIdentidad(){
        return this.carnetDeIdentidad;
    }
    set_carnetDeIdentidad(carnetDeIdentidad:any){
        this.carnetDeIdentidad=carnetDeIdentidad;
    }
    get_nombres(){
        return this.nombres;
    }
    set_nombres(nombres:any){
        this.nombres=nombres;
    }
    get_apellidos(){
        return this.apellidos;
    }
    set_apellidos(apellidos:any){
        this.apellidos=apellidos;
    }
    get_lugarDeNacimiento(){
        return this.lugarDeNacimiento;
    }
    set_lugarDeNacimiento(lugarDeNacimiento:any){
        this.lugarDeNacimiento=lugarDeNacimiento;
    }
    get_fechaDeNacimiento(){
        return this.fechaDeNacimiento;
    }
    set_fechaDeNacimiento(fechaDeNacimiento:any){
        this.fechaDeNacimiento=fechaDeNacimiento;
    }
    get_sexo(){
        return this.sexo;
    }
    set_sexo(sexo:any){
        this.sexo=sexo;
    }
    get_numeroDeContacto(){
        return this.numeroDeContacto;
    }
    set_numeroDeContacto(numeroDeContacto:any){
        this.numeroDeContacto=numeroDeContacto;
    }
    get_parientes(){
        return this.parientes;
    }
    set_parientes(parientes:any){
        this.parientes=parientes;
    }
    get_correoElectronico(){
        return this.correoElectronico;
    }
    set_correoElectronico(correoElectronico:any){
        this.correoElectronico=correoElectronico;
    }
    get_contrasena(){
        return this.contrasena;
    }
    set_contrasena(contrasena:any){
        this.contrasena=contrasena;
    }
    get_nombreDeUsuario(){
        return this.nombreDeUsuario;
    }
    set_nombreDeUsuario(nombreDeUsuario:any){
        this.nombreDeUsuario=nombreDeUsuario;
    }
    get_agendaVirtual(){
        return this.agendaVirtual;
    }
    set_agendaVirtual(agendaVirtual:any){
        this.agendaVirtual=agendaVirtual;
    }
}

File: Proyecto_APK_Movil/app/src/main/java/com/example/proyecto_apk/data/model/Paciente.java
package com.example.proyecto_apk.data.model;

import java.util.Date;

public class Paciente {
  int carnetDeIdentidad;
  String nombres;
  String apellidos;
  String lugarDeNacimiento;
  Date fechaDeNacimiento;
  String sexo;
  int numeroDeContacto;
  String correoElectronico;
  String contrasena;
  String nombreDeUsuario;
  Object parientes;
  String hash;
  Object agendaVirtual;
  String token;

  public Paciente() {
  }

  public Paciente(int carnetDeIdentidad, String nombres, String apellidos, String lugarDeNacimiento, Date fechaDeNacimiento, String sexo, int numeroDeContacto, String correoElectronico, String contrasena, String nombreDeUsuario, Object parientes, String hash, Object agendaVirtual, String token) {
    this.carnetDeIdentidad = carnetDeIdentidad;
    this.nombres = nombres;
    this.apellidos = apellidos;
    this.lugarDeNacimiento = lugarDeNacimiento;
    this.fechaDeNacimiento = fechaDeNacimiento;
    this.sexo = sexo;
    this.numeroDeContacto = numeroDeContacto;
    this.correoElectronico = correoElectronico;
    this.contrasena = contrasena;
    this.nombreDeUsuario = nombreDeUsuario;
    this.parientes = parientes;
    this.hash = hash;
    this.agendaVirtual = agendaVirtual;
    this.token = token;
  }

  public int getCarnetDeIdentidad() {
    return carnetDeIdentidad;
  }

  public void setCarnetDeIdentidad(int carnetDeIdentidad) {
    this.carnetDeIdentidad = carnetDeIdentidad;
  }

  public String getNombres() {
    return nombres;
  }

  public void setNombres(String nombres) {
    this.nombres = nombres;
  }

  public String getApellidos() {
    return apellidos;
  }

  public void setApellidos(String apellidos) {
    this.apellidos = apellidos;
  }

  public String getLugarDeNacimiento() {
    return lugarDeNacimiento;
  }

  public void setLugarDeNacimiento(String lugarDeNacimiento) {
    this.lugarDeNacimiento = lugarDeNacimiento;
  }

  public Date getFechaDeNacimiento() {
    return fechaDeNacimiento;
  }

  public void setFechaDeNacimiento(Date fechaDeNacimiento) {
    this.fechaDeNacimiento = fechaDeNacimiento;
  }

  public String getSexo() {
    return sexo;
  }

  public void setSexo(String sexo) {
    this.sexo = sexo;
  }

  public int getNumeroDeContacto() {
    return numeroDeContacto;
  }

  public void setNumeroDeContacto(int numeroDeContacto) {
    this.numeroDeContacto = numeroDeContacto;
  }

  public String getCorreoElectronico() {
    return correoElectronico;
  }

  public void setCorreoElectronico(String correoElectronico) {
    this.correoElectronico = correoElectronico;
  }

  public String getContrasena() {
    return contrasena;
  }

  public void setContrasena(String contrasena) {
    this.contrasena = contrasena;
  }

  public String getNombreDeUsuario() {
    return nombreDeUsuario;
  }

  public void setNombreDeUsuario(String nombreDeUsuario) {
    this.nombreDeUsuario = nombreDeUsuario;
  }

  public Object getParientes() {
    return parientes;
  }

  public void setParientes(Object parientes) {
    this.parientes = parientes;
  }

  public String getHash() {
    return hash;
  }

  public void setHash(String hash) {
    this.hash = hash;
  }

  public Object getAgendaVirtual() {
    return agendaVirtual;
  }

  public void setAgendaVirtual(Object agendaVirtual) {
    this.agendaVirtual = agendaVirtual;
  }

  public String getToken() {
    return token;
  }

  public void setToken(String token) {
    this.token = token;
  }

  public String getUserId() {
    return "";
  }

  public String getDisplayName() {
    return "";
  }
}
